src/js/controllers/chatController.js

```diff
@@ -352,14 +353,27 @@ module.exports = ViewController.extend({
 
     _getLatestReadTime: function() {
         if (!this.latestReadTs) {
-            this.latestReadTs = parseInt(cookie.parse(document.cookie).sk_latestts || 0);
+            var key = SK_LATEST_TS + '_' + endpoint.appUserId;
```

Comment:
Read state should be tracked by `userId`. `appUserId` can pretty much change at any time